manifests/a/Amazon/WorkspacesClient/5.21.0/Amazon.WorkspacesClient.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Amazon.WorkspacesClient
+PackageVersion: 5.21.0
+InstallerLocale: en-US
+InstallerType: wix
+InstallerSwitches:
+  InstallLocation: APPLICATIONFOLDER="<INSTALLPATH>"
+UpgradeBehavior: install
+ProductCode: '{8BD9A0C2-E73D-4315-BFFD-9199C78BA355}'
+ReleaseDate: 2024-06-19
+AppsAndFeaturesEntries:
+- DisplayVersion: 5.20.0.4722
+  UpgradeCode: '{CB7B8EA8-3D5A-4233-A9CB-31A692E24E62}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi
+  InstallerSha256: 7394FB18F796E44CA4BD94BF25980D1CE25606A39D68478227127B705F2E7631

```

```suggestion
  InstallerSha256: 493D9AAD3F793239C0B2110180A6BA1E2C59C5D852B4521AD5A8D7C26DED0DCA
```
Apologies for the noisy review.